### ISSUE TITLE
[PRE REVIEW]: BayesMFSurv: An R Package to Estimate Bayesian Split-Population Survival Models With (and Without) Misclassified Failure Events

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#2069]
**Submitting author:** @Nicolas-Schmidt (<a href="http://orcid.org/0000-0001-5083-5792">Nicolás Schmidt</a>)
**Repository:** <a href="https://github.com/Nicolas-Schmidt/BayesMFSurv" target ="_blank">https://github.com/Nicolas-Schmidt/BayesMFSurv</a>
**Version:** 0.2.0
**Editor:** @marcosvital
**Reviewers:** @alletsee, @andybega

**Author instructions**

Thanks for submitting your paper to JOSS @Nicolas-Schmidt. **Currently, there isn't an JOSS editor assigned** to your paper.

@Nicolas-Schmidt if you have any suggestions for potential reviewers then please mention them here in this thread (without tagging them with an @). In addition, [this list of people](https://bit.ly/joss-reviewers) have already agreed to review for JOSS and may be suitable for this submission (please start at the bottom of the list).

**Editor instructions**

The JOSS submission bot @whedon is here to help you find and assign reviewers and start the main review. To find out what @whedon can do for you type:

```
@whedon commands
```

**Comment by whedon** [comment]
Hello human, I'm @whedon, a robot that can help you with some common editorial tasks.

For a list of things I can do to help you, just type:

```
@whedon commands
```

For example, to regenerate the paper pdf after making changes in the paper's md or bib files, type:

```
@whedon generate pdf
```

**Comment by whedon** [comment]
```
Software report (experimental):

github.com/AlDanial/cloc v 1.84  T=0.06 s (283.2 files/s, 41640.5 lines/s)
-------------------------------------------------------------------------------
Language                     files          blank        comment           code
-------------------------------------------------------------------------------
R                               10            181            686            913
Markdown                         3             51              0            256
TeX                              1             20              0            181
C++                              3             17             19            155
Rmd                              1             54             72             42
-------------------------------------------------------------------------------
SUM:                            18            323            777           1547
-------------------------------------------------------------------------------


Statistical information for the repository '2069' was gathered on 2020/02/04.
The following historical commit information, by author, was found:

Author                     Commits    Insertions      Deletions    % of changes
Nicolas-Schmidt                  4           197              6          100.00

Below are the number of rows from each author that have survived and are still
intact in the current revision:

Author                     Rows      Stability          Age       % in comments
Nicolas-Schmidt             191           97.0          0.0                9.95
```

**Comment by whedon** [comment]
```
Reference check summary:

OK DOIs

- 10.1038/sj.bjc.6601118 is OK
- 10.32614/RJ-2017-056 is OK
- 10.1002/sim.687 is OK
- 10.1214/aos/1056562461 is OK
- 10.1111/j.1468-2478.2007.00469.x is OK
- 10.1213/ANE.0000000000003653 is OK
- 10.1111/j.1468-2478.2012.00719.x is OK
- 10.1017/pan.2019.6 is OK
- 10.1111/0020-8833.00137 is OK

MISSING DOIs

- https://doi.org/10.1080/00401706.1998.10485509 may be missing for title: Survival analysis with long-term survivors

INVALID DOIs

- None
```

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.02069/joss.02069/10.21105.joss.02069.pdf)

**Comment by kthyng** [comment]
@marcosvital Could you edit this submission?

**Comment by marcosvital** [comment]
Hi, @kthyng ! Yes, I can handle this one.

**Comment by danielskatz** [comment]
@whedon assign @marcosvital as editor

**Comment by whedon** [comment]
OK, the editor is @marcosvital

**Comment by danielskatz** [comment]
👋 @marcosvital - note that you can issue this whedon command as well - there's no need to wait for the AEiC on duty to do it, once you agree that you will edit.

**Comment by marcosvital** [comment]
Ah, didn't know that, thanks @danielskatz ! :)

Hi, @Nicolas-Schmidt, I'll be the editor of your submission.

Please take a look at the article proof generated above, and check if everything seems to be ok.

On the meantime, we can find reviewers for you submission. We usually ask the authors for suggestions, so let me know if you have anyone in mind that could do this. You can check [this list of people](https://bit.ly/joss-reviewers) that have agreed to review for JOSS, and see if any of them may be suitable to review this submission - but feel free to suggest other people as well, ok?

And let me know if you need any help finding reviewers if you don't have anyone in mind.

**Comment by Nicolas-Schmidt** [comment]
@whedon generate pdf

**Comment by whedon** [comment]
[ :point_right: Check article proof :page_facing_up: :point_left: ](https://github.com/openjournals/joss-papers/blob/joss.02069/joss.02069/10.21105.joss.02069.pdf)

**Comment by Nicolas-Schmidt** [comment]
Hi @marcosvital, 
From the list of reviewers I suggest: alletsee o rdrr1990.
Thanks!

**Comment by Nicolas-Schmidt** [comment]
Hi @marcosvital,
From the list of reviewers I suggest: alletsee or rdrr1990.
Thanks!

**Comment by Nicolas-Schmidt** [comment]
Hi @marcosvital,
We would like to suggest an additional reviewer that is our first preference: Andy Beger (git user: andybega).

Thanks!

**Comment by Sergio-Bejar** [comment]
@whedon would it be possible to get an update on the status of our paper?

**Comment by whedon** [comment]
I'm sorry human, I don't understand that. You can see what commands I support by typing:

```
@whedon commands
```

**Comment by Sergio-Bejar** [comment]
@danielskatz @kthyng @marcosvital , would it be possible to get an update on the status of our paper?

**Comment by danielskatz** [comment]
It's the job of the editor (@marcosvital) to find reviewers, possibly with your help/suggestions.  That's where we are at this point.  Perhaps @marcosvital can say more about where he is, in case he's also been emailing potential reviewers?

**Comment by Sergio-Bejar** [comment]
@danielskatz , thanks for your response.  I look forward to hearing from @marcosvital .

**Comment by Sergio-Bejar** [comment]
@danielskatz @kthyng, apologies for bugging you about this submission.   But there's been no feedback/information from the assigned editor in the past 14 days -specifically since we proposed a set of potential reviewers.-   Other submissions assigned to the same editor seem to have the same issue (no response).  Is it possible to assign the piece to a different editor?  If not, is there anything we can do to help speed up (or at least move to the review stage) process?  

Again, apologies for the multiple messages and thanks for your time.

**Comment by marcosvital** [comment]
Hi, @Sergio-Bejar ! I'm terribly sorry about the delayed reply and for not being able to check in to follow your submission and keep it moving - I was unable to work on this over the last couple of weeks, and should have let the rest of the team know and assigned someone else. I'm truly sorry about this, and will note let that happen again.

I'll follow up calling the potential reviewers, ok, and keep constantly checking the status of your submission. But please let us know if you don't feel comfortable with me as an editor from this point on.

**Comment by marcosvital** [comment]
Dear @andybega, @alletsee and @rdrr1990:

Would you be willing to review this submission for JOSS?
We carry out our checklist-driven reviews here in GitHub issues and follow these guidelines: https://joss.readthedocs.io/en/latest/review_criteria.html

This is still a pre-review, once there are enough reviewers, we will move on to a new issue where the review will take place.

If you are not able, please let us know if you have recommendations for other reviewers.

**Comment by andybega** [comment]
Hi @marcosvital, I’m on vacation until 16 March. Most likely I would not be able to do a review until that week. If that’s ok, I can do it.

**Comment by alletsee** [comment]
Hi @marcosvital,
Thanks for notifying me. I would be willing to review this submission.
I'm working on several things in parallel at the moment, so I might not be able to review it right away. Would a time frame of 2-3 weeks be okay?

**Comment by marcosvital** [comment]
Thank you, @andybega and @alletsee!

@Nicolas-Schmidt and @Sergio-Bejar , are you ok with waiting 2-3 weeks for the reviews to happen? I know that my absence made the whole process take much more time than it should have, so please let me know if you would rather let me try to find other reviewers.

**Comment by Sergio-Bejar** [comment]
@marcosvital, @andybega, @alletsee, thanks for your response.  @marcosvital, we are fine waiting 2-3 weeks for the reviews.  We look forward to receiving comments/suggestions from the reviewers.  Regards!

**Comment by Sergio-Bejar** [comment]
Hi @marcosvital, would it be possible to move to the next stage (i.e. review) since at least 2 potential reviewers accepted the request a couple of weeks ago?
Thanks!

**Comment by arfon** [comment]
Dear authors and reviewers

We wanted to notify you that in light of the current COVID-19 pandemic, JOSS has decided to suspend submission of new manuscripts and to handle existing manuscripts (such as this one) on a "best efforts basis". We understand that you may need to attend to more pressing issues than completing a review or updating a repository in response to a review. If this is the case, a quick note indicating that you need to put a "pause" on your involvement with a review would be appreciated but is not required.

Thanks in advance for your understanding.

_Arfon Smith, Editor in Chief, on behalf of the JOSS editorial team._

**Comment by alletsee** [comment]
Hi @Sergio-Bejar @marcosvital 

I would still be willing and able to review the package within the next 2 weeks approximately. From my perspective, we could open the Github issue for the checklist-based review.

**Comment by marcosvital** [comment]
Hey, everyone!

@Sergio-Bejar , yes, we can move on. I was still expecting a possible response from a third reviewer, but that's not very important.

@alletsee and @andybega , thank you again for your response. We will move on to a different repository, and this one will be automatically closed.

**Comment by marcosvital** [comment]
@whedon start review

**Comment by whedon** [comment]
It looks like you don't have an editor and reviewer assigned yet so I can't start the review. Try one or more of these commands:

```
# Change editorial assignment
@whedon assign @username as editor

# Assign a GitHub user as the sole reviewer of this submission
@whedon assign @username as reviewer

# Add a GitHub user to the reviewers of this submission
@whedon add @username as reviewer
```

**Comment by marcosvital** [comment]
@whedon assign @alletsee as reviewer

**Comment by whedon** [comment]
OK, @alletsee is now a reviewer

**Comment by marcosvital** [comment]
@whedon add @andybega as reviewer

**Comment by whedon** [comment]
OK, @andybega is now a reviewer

**Comment by marcosvital** [comment]
@whedon start review

**Comment by whedon** [comment]
OK, I've started the review over in https://github.com/openjournals/joss-reviews/issues/2164.
